packages/ast-spec/src/declaration/TSModuleDeclaration/fixtures/namespace-nested-once-declare/fixture.ts

```diff
@@ -1 +1 @@
-declare namespace abd.def {}
```

Comment:
Oh, we'll want to not have this...

Reply:
It seems that this issue was resolved in `prettier@3.0.2` https://github.com/prettier/prettier/blob/main/CHANGELOG.md#print-declare-and-export-keywords-for-nested-namespace-15249-by-sosukesuzuki

Reply:
Nice!